packages/widget-chat/src/actions/conversation.js

```diff
@@ -28,8 +45,21 @@ export function createConversationWithUser(userId, spark) {
     spark.conversation.create({
       participants: [userId]
     }, {
-      activitiesLimit: 10
+      latestActivity: true,
+      activitiesLimit: 30
```

Comment:
I was thinking just a simple `config.js` file with an options object. That way we don't have "magic strings" and numbers in the app.